fenix/app/src/test/java/org/mozilla/fenix/tabstray/DefaultTabsTrayControllerTest.kt

```diff
@@ -865,6 +865,34 @@ class DefaultTabsTrayControllerTest {
         verify { controller.handleNavigateToBrowser() }
     }
 
+    @Test
+    fun `GIVEN a tab is selected WHEN handleTabDeletion is called on last private tab THEN that private tab is removed and an undo snackbar is shown and original tab is still displayed`() {
+        val currentTab = TabSessionState(content = ContentState(url = "https://simulate.com", private = false), id = "currentTab", createdAt = 11)
+        val privateTab = TabSessionState(content = ContentState(url = "https://mozilla.com", private = true), id = "privateTab", createdAt = 33)
```

Comment:
nit: the `createdAt` parameter might be able to be removed (can't remember if it's optional)

Reply:
ah I removed the `createdAt` parameter. thank you!